UpdateDependencies.ps1

```diff
@@ -22,6 +22,16 @@ param(
 $LatestVersion = Invoke-WebRequest $VersionFileUrl -UseBasicParsing
 $LatestVersion = $LatestVersion.ToString().Trim()
 
+if ($DirPropsVersionElements -contains 'CoreClrExpectedPrerelease')
+{
+    # Also get list of all package versions, relative to the given prerelease version url.
+    $LatestPackagesListUrl = $VersionFileUrl -Replace 'Latest.txt', 'Latest_Packages.txt'
+    $LatestPackagesList = Invoke-WebRequest $LatestPackagesListUrl -UseBasicParsing
+    $LatestCoreCLRPackage = $LatestPackagesList -split "`n" | ?{ $_.StartsWith('Microsoft.NETCore.Runtime.CoreCLR') }
+    $LatestCoreCLRVersion = ($LatestCoreCLRPackage -split ' ')[1].Trim()
+}
```

Comment:
@eerhardt Just noting that this is using the `Latest_Packages.txt` file to find the full version of the CoreCLR runtime specifically. (And that I'll be replacing this with the versiontools functionality soon. Don't want this to diverge from other repos for any length of time.)